x-pack/plugins/enterprise_search/server/integrations.ts

```diff
@@ -850,6 +885,42 @@ export const registerEnterpriseSearchIntegrations = (
       shipper: 'enterprise_search',
       isBeta: false,
     });
+    customIntegrations.registerCustomIntegration({
+      id: 'jira_data_center',
+      title: i18n.translate(
+        'xpack.enterpriseSearch.integrations.connectors.jiraDataCenterCloudTitle',
+        {
+          defaultMessage: 'Jira Data Center',
+        }
+      ),
+      description: i18n.translate(
+        'xpack.enterpriseSearch.integrations.connectors.jiraDataCenterDescription',
+        {
+          defaultMessage: 'Search over your content on Jira Data Center.',
+        }
+      ),
+      categories: [
+        'enterprise_search',
+        'elastic_stack',
+        'custom',
+        'connector',
+        'connector_client',
+        'jira_data_center',
```

Comment:
Same here, I don't think it makes sense for`jira_data_center` to be a category

Reply:
Ah, makes sense, thanks, will fix now!